docs/open-source-tizen/tools/gbs/gbs-changelog.md

```diff
@@ -1,12 +1,14 @@
 # gbs changelog
 
-Subcommand changelog is used to generate changelog file in ./packaging dir. It is mostly used for creating a changelog before submitting code. You can get the usage of subcommand changelog by using '$ gbs changelog --help'
+Use the `changelog` subcommand to generate a change log file in the `./packaging` directory.
+
+The command is mostly used for creating a change log before submitting code. For command usage details, enter:
 
 ```bash
 $ gbs changelog --help
 ```
 
-Examples:
+Change log examples:
```

Comment:
Is change log redundant? The example is located in changelog section.

Although you add "Change log", you must write changelog because it is an option of gbs command.

Reply:
This needs to be synched with the issue below.
We will fix it to "Examples of command usage:" or "Examples:", depending on your preference.